apps/src/applab/dropletConfig.js

```diff
@@ -43,6 +43,8 @@ module.exports.blocks = [
   {func: 'dropdown', parent: api, category: 'UI controls', paletteParams: ['id','option1','etc'], params: ['"id"', '"option1"', '"etc"'] },
   {func: 'getText', parent: api, category: 'UI controls', paletteParams: ['id'], params: ['"id"'], dropdown: { 0: function () { return Applab.getIdDropdown(); } }, type: 'value' },
   {func: 'setText', parent: api, category: 'UI controls', paletteParams: ['id','text'], params: ['"id"', '"text"'], dropdown: { 0: function () { return Applab.getIdDropdown(); } } },
+  {func: 'getValue', parent: api, category: 'UI controls', paletteParams: ['id'], params: ['"id"'], dropdown: { 0: function () { return Applab.getIdDropdown(); } }, type: 'value' },
+  {func: 'setValue', parent: api, category: 'UI controls', paletteParams: ['id','value'], params: ['"id"', '0'], dropdown: { 0: function () { return Applab.getIdDropdown(); } } },
```

Comment:
Since `getValue` (soon to be `getNumber`) is only usable with the slider for now, you could filter the element ID dropdown to only include slider elements, and we can expand it later when we extend `getNumber`.  You just need to pick an appropriate jQuery selector, probably

```javascript
dropdown: { 0: function () { return Applab.getIdDropdown('input[type=“range”]'); }
```

[Here's an example from the Chart API.](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/applab/dropletConfig.js#L96)